acm/importCertificate.ts

```diff
@@ -33,6 +34,12 @@ const importCertificate = async (
       Options.Defaults.override({ log })
     )
 
+    const keySize = new NodeRSA(PrivateKey).getKeySize()
+
+    if (keySize > 4096) {
```

Comment:
The condition will be false for keySize equal to 4096. You likely mean greater or equal.